android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -650,6 +663,26 @@ public void pickFeature(float posX, float posY) {
             nativePickFeature(mapPointer, posX, posY, featurePickListener);
         }
     }
+    /**
+     * Set a listener for label pick events
+     * @param listener Listener to call
+     */
+    public void setLabelPickListener(LabelPickListener listener) {
+        labelPickListener = listener;
+    }
+
+    /**
+     * Query the map for labeled features at the given screen coordinates; results will be returned
+     * in a callback to the object set by {@link #setLabelPickListener(LabelPickListener)}
+     * @param posX The horizontal screen coordinate
+     * @param posY The vertical screen coordinate
+     */
+    public void pickLabels(float posX, float posY) {
```

Comment:
Currently there aren't any calls to `pickFeatureAt(...)` in the Android SDK or Eraser Map code yet the `FeaturePickListnener.onFeaturePick(...)` callback still gets invoked on single tap. Is that going to change with this pull request?